holder.yml

```diff
@@ -10,10 +10,106 @@ info:
     name: GitHub Source Code
     url: https://github.com/w3c-ccg/vc-api
 paths:
+  /credentials/{id}:
+    get:
+      tags:
+       - Credentials
+      summary: Gets a credential or verifiable credential by ID
+      operationId: getCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Credential retrieved
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: "./components/Credential.yml#/components/schemas/Credential"
+                  - $ref: "./components/VerifiableCredential.yml#/components/schemas/VerifiableCredential"
+        "400":
+          description: Bad Request
+        "401":
+          description: Not Authorized
```

Comment:
are we going to define authorization? and test it?

Reply:
> are we going to define authorization? and test it?

The test suites will need to have some authz mechanism in them because the production software that all implementers are implementing are all (AFAIK) requiring authz. If we have interop on the test suites, the spec will have to reflect reality and document how authz is being performed in an interoperable fashion.